modules/nf-core/wittyer/meta.yml

```diff
@@ -0,0 +1,66 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "wittyer"
+description: A large variant benchmarking tool analogous to hap.py for small variants.
+keywords:
+  - structural-variants
+  - benchmarking
+  - vcf
+tools:
+  - "wittyer":
+      description: "Illumina tool for large variant benchmarking"
+      homepage: "https://github.com/Illumina/witty.er"
+      documentation: "https://github.com/Illumina/witty.er"
+      tool_dev_url: "https://github.com/Illumina/witty.er"
+      licence: "['BSD-2']"
```

Comment:
This is the licence that is indicated on the witty.er page also somehow throws a linting error.

Reply:
licence: ['BSD-2']